medusa/helpers.py

```diff
@@ -1021,7 +1021,7 @@ def validate_show(show, season=None, episode=None):
         if show.dvdorder != 0:
             indexer_api_params['dvdorder'] = True
 
-        t = indexerApi(show.indexer).indexer(**indexer_api_params)
+        t = show.get_indexer_api()
```

Comment:
One letter variables should be avoided except in cases where a one letter argument is the most clear (for example using x and y as properties of a cartesian coordinate class)